src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.cs

```diff
@@ -47,6 +47,7 @@ public object ScrollOwner
 			}
 		}
 		#endregion
+		public bool ForceChangeToCurrentView { get; set; }
```

Comment:
```suggestion
		internal bool ForceChangeToCurrentView { get; set; }
```

Reply:
Because of the Interface `IScrollContentPresenter` (https://github.com/ajpinedam/uno/blob/dev/add.flipview.support/src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/IScrollContentPresenter.wasm.cs#L11), I was unable to apply this change. Any suggestion?

Reply:
you'll need to explicitly implement the interface property instead:

```suggestion
		bool IScrollContentPresenter.ForceChangeToCurrentView { get; set; }
```

Reply:
@davidjohnoliver  Explicitly implementing it, it's not working either. Let's connect

Reply:
Done / Completed